### PR TITLE
Handle clrdbg to vsdbg rename

### PR DESCRIPTION
**Pull request by wesrupert**
@gregg-miskelly @chuckries @wardengnaw

### CODE REVIEW

File: tools/InstallToVSCode/InstallToVSCode.cmd [hunk 1]
```diff
@@ -82,32 +82,38 @@ dotnet publish -o %DESTDIR%
 if NOT "%ERRORLEVEL%"=="0" echo "ERROR: 'dotnet publish' failed." & exit /b -1
 popd
 
-pushd %DESTDIR%
-ren dummy.exe OpenDebugAD7.exe
-if NOT "%ERRORLEVEL%"=="0" echo ERROR: Unable to rename OpenDebugAD7.exe???& exit /b -1
+pushd %destdir%
```
Comment (WardenGnaw) [inline tools/InstallToVSCode/InstallToVSCode.cmd]: Accidental 'gu' of these three lines?

Comment (wesrupert) [inline tools/InstallToVSCode/InstallToVSCode.cmd]: Fixed.

File: tools/InstallToVSCode/InstallToVSCode.cmd [hunk 2]
```diff
@@ -92,22 +92,28 @@ for %%f in (xunit.console.netcore.exe) do call :InstallFile "%OpenDebugAD7BinDir
...
+for %%f in (%VSDBGBITSDIR%\version.txt) do call :InstallFile "%%f"
+for /D %%d in (%VSDBGBITSDIR%\*) do (
     echo.
-    echo Installing clrdbg bits from %%d... to %%~nd
+    echo Installing vsdbg bits from %%d... to %%~nd
     if NOT exist "%DESTDIR%\%%~nd" mkdir "%DESTDIR%\%%~nd
     for %%f in (%%d\*.dll) do call :InstallFile "%%f" %%~nd\
 )
 
+REM Rename vsdbg back to clrdbg
```
Comment (chuckries) [inline tools/InstallToVSCode/InstallToVSCode.cmd]: is it worth having this part behind an option so we can toggle whether to rename it or not?

File: tools/InstallToVSCode/InstallToVSCode.sh [hunk 3]
```diff
@@ -237,40 +237,44 @@ mv "$DESTDIR/dummy" "$DESTDIR/OpenDebugAD7"
...
     fi
-        
+
     for dll in $(ls $directory/*.dll); do
         install_file "$dll" "$directory_name/"
     done
 done
 
+# Rename vsdbg back to clrdbg
+mv "$DESTDIR/vsdbg" "$DESTDIR/clrdbg"
```
Comment (chuckries) [inline tools/InstallToVSCode/InstallToVSCode.sh]: same here.

**Pull request merged**